proto/redpanda/api/dataplane/v1alpha1/kafka_connect.proto

```diff
@@ -432,6 +579,13 @@ service KafkaConnectService {
     option (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_operation) = {
       summary: "Resume connector"
       description: "Resume a paused connector and its tasks, resumes message processing after the connector has been paused. It won't do nothing if the connector is not paused."
+      responses: {
+        key: "202"
+        value: {
+          description: "Accepted"
```

Comment:
```suggestion
          description: "Resume request accepted"
```
I think we could also just say "Request accepted" if we want to be even more economical with the message

Same question here regarding how the user can check whether the connector has resumed